### DISCUSSION TITLE
Making vanilla site

### DISCUSSION THREAD

**Discussion by Brisargr** [leokhoa/laragon#573] [Q&A] [answered]
I find Paragon the most useful of all the stand-alone website hosts, but find that whenever I generate a new site there are a number of things that have always to be changed.  Is it possible to generate a "vanilla" site that can be used each time?

Each time the version of Wordpress needs to be updated 
Screen options in Wordpress need to be removed
Pages and posts supplied with new Wordpress need to be deleted
In Wordpress settings for media, all sizes need to be removed and stored in monthly order removed

Is it possible to make these changes and make the resultant site usable for every site?

Thanks in anticipation

Brian

**Comment by dustinatx** [comment, marked as answer]
You could create a site with WordPress how you want it, then clone it. From the menu, go to Quick App > Clone.

**Reply by Brisargr** [reply]
Many thanks that works brilliantly and saves a lot of time
